spec/cloudsearchable/cloudsearchable_spec.rb

```diff
@@ -7,14 +7,15 @@
   it 'can describe an index that returns ids for the class type' do
     test_index = clazz.cloudsearch_index
     test_index.should be_kind_of(Cloudsearchable::Domain)
-    test_index.fields.should have(4).items #3 explicit + 1 for the id of the object
+    # test_index.fields.should have(4).items #3 explicit + 1 for the id of the object
```

Comment:
Miss this comment. Sorry, I have removed the comment. I didn't mean to commit with the comment. I only had it during testing the change as reference.